src/main/java/com/kodilla/ecommercee/repository/ProductRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.Product;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+
+@Transactional
+@Repository
+public interface ProductRepository extends CrudRepository<Product, Long> {
```

Comment:
Don't you think that we should create methods here? In the next task we will need them to tests.